share/spack/gitlab/cloud_pipelines/stacks/hep/spack.yaml

```
@@ -0,0 +1,64 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: when_possible
+
+  packages:
+    all:
+      require: '%gcc target=x86_64_v3'
+      providers:
+        blas: [openblas]
+        mpi: [mpich]
+        tbb: [intel-tbb]
+      variants: +mpi
+
+    # Mark geant4 data as external to prevent wasting bandwidth on GB-scale files
+    geant4-data:
+      externals:
+        - spec: geant4-data@11.3.0
+          prefix: /usr
+        - spec: geant4-data@11.2.2
+          prefix: /usr
+        - spec: geant4-data@11.2.0
+          prefix: /usr
+        - spec: geant4-data@11.1.0
+          prefix: /usr
+        - spec: geant4-data@11.0.0
+          prefix: /usr
+
+  specs:
+  # CPU
+  - acts +analysis +binaries +dd4hep +edm4hep +examples +fatras +geant4 +hepmc3 +podio +pythia8 +python +tgeo
+  - dd4hep +ddalign +ddcad +ddcond +dddetectors +dddigi +ddeve +ddg4 +ddrec +edm4hep +hepmc3 +lcio +utilityapps +xercesc
+  - delphes +pythia8
+  - edm4hep
+  - fastjet
+  - fjcontrib
+  - gaudi +aida +examples +heppdt +xercesc
+  - geant4 +opengl +qt +threads ~vtk
+  - hepmc
+  - hepmc3 +interfaces +protobuf +python +rootio
+  #- herwig3 +njet +vbfnlo  # Note: herwig3 fails to find evtgen
+  - lcio -examples ~jar +rootdict  # Note: lcio +examples ^ncurses -termlib, which leads to conflicts
+  - lhapdf +python
+  - madgraph5amc
+  - opendatadetector
+  - podio +rntuple +sio
+  - py-uproot +lz4 +xrootd +zstd
+  - pythia8 +evtgen +fastjet +hdf5 +hepmc +hepmc3 +lhapdf ~madgraph5amc +python +rivet -root # pythia8 and root circularly depend
+  - rivet hepmc=3
+  - root +davix +dcache +examples +fftw +fits +fortran +gdml +graphviz +gsl +http +math +minuit +mlp +mysql +opengl ~oracle ~postgres +pythia6 +pythia8 +python +r +roofit +root7 ~shadow +spectrum +sqlite +ssl +tbb +threads +tmva +unuran +vc +vdt +veccore +webgui +x +xml +xrootd
```

`root ~oracle` pins to `root@:6.30` due to removed variant.

```suggestion
  - root +davix +dcache +examples +fftw +fits +fortran +gdml +graphviz +gsl +http +math +minuit +mlp +mysql +opengl ~postgres +pythia6 +pythia8 +python +r +roofit +root7 ~shadow +spectrum +sqlite +ssl +tbb +threads +tmva +unuran +vc +vdt +veccore +webgui +x +xml +xrootd
```